bingo/bingo-elastic/java/pom.xml

```diff
@@ -53,7 +53,7 @@
         <dependency>
             <groupId>org.elasticsearch.client</groupId>
             <artifactId>elasticsearch-rest-high-level-client</artifactId>
-            <version>7.16.2</version>
```

Comment:
move java changes into a separate PR

Reply:
agreed

Reply:
That issue was resolved at [9508247](https://github.com/epam/Indigo/commit/9508247149d1b1dbf455e4cb9340566e8d2d9758)